mitodl/bootcamp-ecommerce
# Add results_url from Jobma webhook

#### What are the relevant tickets?
None

#### What's this PR do?
Adds a field to store the `results_url` from the Jobma webhook

#### How should this be manually tested?
I would skip manual testing since it's complicated to set up. I ran through an interview workflow and you should be able to see the results_url data in the PR build database under the `Interview` model (there is only one right now).
